packages/commands/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@dhis2/cypress-helpers",
```

Comment:
I think this should probably match the workspace directory, either `@dhis2/cypress-helpers` in `packages/helpers` or `@dhis2/cypress-commands` in `packages/commands`

Reply:
I've renamed both the package name as well as the folder to `cypress-commands`